sourcemash/frontend/templates/help.tpl

```diff
@@ -0,0 +1,65 @@
+<div class="help-fixed-action-btn">
+  <span class="btn-floating btn-large">?</span>
+</div>
+
+<div id="help-modal" class="modal">
+  <div class="modal-content section no-pad-bot">
+    <div class="container">
+      <div class="row">
+
+        <h2 class="col s12 light center header flow-text">Confused?</h2>
```

Comment:
Either works

Alex Gerstein
gerstein.alex@gmail.com




On Wed, May 20, 2015 at 2:39 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/templates/help.tpl:
> @@ -0,0 +1,65 @@
> +<div class="help-fixed-action-btn">
> +  <span class="btn-floating btn-large">?</span>
> +</div>
> +
> +<div id="help-modal" class="modal">
> +  <div class="modal-content section no-pad-bot">
> +    <div class="container">
> +      <div class="row">
> +
> +        <h2 class="col s12 light center header flow-text">Confused?</h2>

Logo or type?



—
Reply to this email directly or view it on GitHub.